src/pages/PlantDatabase.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+export const PlantDatabase = () => {
+  const [plantSpeciesList, setPlantSpeciesList] = useState([]);
+  useEffect(() => {
+    for (let i = 1; i < 4; i += 1) {
```

Comment:
What does this line do?

Reply:
it's supposed to return results for pages 1,2,3 of the API - not working though, I have a stack overflow question open about it https://stackoverflowteams.com/c/technigo/questions/4966